reframe/frontend/cli.py

```diff
@@ -751,7 +749,19 @@ def print_infoline(param, value):
                         f'skipping...'
                     )
                     printer.verbose(str(e))
+            elif d.startswith('+'):
+                try:
+                    rt.modules_system.searchpath_add(d[1:])
+                except errors.EnvironError as e:
+                    printer.warning(
+                        f'could not use module path {d} correctly; '
+                        f'skipping...'
+                    )
+                    printer.verbose(str(e))
             else:
+                rt.modules_system.searchpath_remove(
+                    *rt.modules_system.searchpath
+                )
```

Comment:
I don't understand this very well. Wouldn't you simply set the `MODULEPATH` here?

Reply:
I wanted to manipulate the module path without setting `MODULEPATH` directly but through the modules api.

Reply:
As discussed offline, this is fine.